src/index.js

```diff
@@ -41,6 +41,21 @@ async function isWithinRange(forwardedFor, fastlyPublicIps, whitelistedIps = '')
     .some((myranges) => (range.isRange ? range.inRange(ip, myranges) : range === ip)));
 }
 
+function getEmbedKind(domainList) {
+  // remove first level domain
+  domainList.pop();
+  const embedKind = [];
+  domainList
+    .filter((domain) => domain !== 'www' && domain !== 'co')
```

Comment:
might need to extend this list at one point:
https://en.wikipedia.org/wiki/Subdomain#Uses